### ISSUE TITLE
[FVT]new xcattest usage concerns

### ISSUE THREAD

**Issue by zet809** [xcat2/xcat-core#3138]
When using xcattest, I created a work directory for myself, and put the cases, bundles here.

The question:
1. How can I specify my own bundle file in my working directory?
```
root@c910f03c05k10:/opt/xcat/share/xcat/tools/autotest/testcase/zettest# xcattest -f zet.conf -b ./zet000
There isn't file /opt/xcat/bin/../share/xcat/tools/autotest/bundle//./zet000
```
2. In my bundle "zet000", 2 cases were added, what happened to my first case?
```
root@c910f03c05k10:/opt/xcat/share/xcat/tools/autotest/testcase/zettest# cat zet000
case1
case2
root@c910f03c05k10:/opt/xcat/share/xcat/tools/autotest/testcase/zettest# xcattest -f zet.conf -b /opt/xcat/share/xcat/tools/autotest/testcase/zettest/zet000
xCAT automated test started at Thu May 25 02:41:14 2017
******************************
loading Configure file
******************************
Varible:
    CN = c01f02x03v04
******************************
Initialize xCAT test environment by definition in configure file
******************************
Detecting: OS = Linux
Detecting: ARCH = Unknown
Detecting: HCP = Unknown
******************************
loading test cases
******************************
Miss attribute: case2 miss attribute SN           ===> What happened to my case1?

```

**Comment by hu-weihua** [comment]
@zet809, this issue has been fixed by PR #4279,  could you please verify this issue?   You can refer to #4279 to get the output of fix. thanks.

**Comment by hu-weihua** [comment]
@zet809  . I have not received any feedback for this issue up to now, I suppose the fix has worked.
Close this issue first, if you find any problem in future, you can reopen this issue again, thanks.
